dashboard/v1.1/src/pages/Ping/index.tsx

```diff
@@ -1,7 +1,151 @@
-import React, { FC } from 'react';
+import React, { FC, useState, useEffect } from 'react';
+import { useFetch } from '../../utils/useFetch';
+import { service_states, HOST_IP } from '../../utils/types';
+import { Spinner } from 'reactstrap';
+import { Card, CardContent, TextField } from '@material-ui/core';
+import {
+  RoutesSummary,
+  QueryResponse,
+  chartData,
+  APIQueryResponse
+} from '../../utils/queryTypes';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import Alert from '@material-ui/lab/Alert';
+import { formatTime } from '../../utils/brt';
+import Ping from '../Monitoring/Ping';
 
-const Ping: FC<{}> = () => {
-  return <div>Ping Page</div>;
+interface showChartsDataParam {
+  pingMin: chartData[];
+  pingMean: chartData[];
+  pingMax: chartData[];
+}
+
+const PingModule: FC<{}> = () => {
```

Comment:
`PingModule` => `Ping`

Reply:
A file and component named Ping already exist within the `Monitoring` folder, so have named it as PingModule to avoid the confusion.